packages/design-system-docs/src/pages/components/DateField/DateField.example.jsx

```diff
@@ -34,16 +34,17 @@ class ControlledDateField extends React.PureComponent {
 }
 
 ReactDOM.render(
-  <Fragment>
+  <div>
```

Comment:
Why was this changed to a div? Couldnt we keep it as a Fragment, or as the Fragment short syntax (i.e. `<>`)?

Reply:
When I try to commit is get an eslint error, on Fragment. 
```
error  'Fragment' is defined but never used  no-unused-vars
```

Reply:
The error message is saying you defined `Fragment` but didnt use it, so all you need to do is remove the unused `Fragment` import

Reply:
When i remove the Fragment import I but keep the `<Fragment>` I get this error. This is why I changed to use the `<div>`
```
 error  'Fragment' is not defined  react/jsx-no-undef
```

Reply:
You want to use the Fragment shorthand instead of `Fragment`
```
<>
   <DateField> 
</>
```
you can find other examples of this if you search for `<>`